src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2071,19 +2069,21 @@ class CgroupUtils:
                 pbs.logmsg(pbs.EVENT_DEBUG2,
                            "%s: Job %s contains defunct process" %
                            (caller_name(), jobid))
-                raise CgroupProcessingError("Job contains defunct process")
+                # Use a list comprehension to remove all instances of the
+                # number 1
+                pids = [x for x in pids if x != 1]
         # check pids to make sure that they are owned by the job owner
         if pbs_version[0] >= 13:
```

Comment:
True, but the cgroups hook is meant to run in a wide variety of configurations. It is likely an administrator will replace their existing cgroups hook with a newer one, even if they are running a 12.x version.